types/templates.go

```diff
@@ -412,6 +413,10 @@ type ValidatorPageData struct {
 	AddValidatorWatchlistModal               *AddValidatorWatchlistModal
 	NextWithdrawalRow                        [][]interface{}
 	ConsolidationRequests                    []*FrontendConsolidationRequest
+	ExecutionConsolidations                  []*FrontendExecutionConsolidationRequest
+	MoveToCompoundingRequests                *FrontendMoveToCompoundingRequest
```

Comment:
```suggestion
	MoveToCompoundingRequest                *FrontendMoveToCompoundingRequest
```
(+ rename all references)